docbook/src/docbkx/autoscale-devguide.xml

```diff
@@ -461,12 +458,39 @@ is external so it's handy for Getting Started to share -->
         <title>Auto Scale core concepts</title>
         <para>To use the &PROD; API effectively, you should understand
             several key concepts: </para>
-        <section xml:id="Glossary">
-            <title>Glossary</title>
-            <!-- Glossary section, defining basic terms,
-is external so it's handy for Getting Started to share -->
-            <xi:include href="chapters/glossary.xml" xpointer="element(Glossary-000)"/>
-            <!-- -->
+        <section xml:id="How_Auto_Scale_Works">
+            <title>How Auto Scale Works</title>
+            <para>Rackspace &PROD; is an API-based tool that automatically scales resources in response to
+                an increase or decrease in overall workload based on user-defined thresholds.</para>
+            <para>&PROD; functions by linking these services:
+                <itemizedlist>
+                    <listitem>
+                        <para>&PROD; API, described in this guide.</para>
```

Comment:
yes, &PROD; is "Auto Scale". There was a bit of a chat on slack about this last night and i had the impression that we wanted to emphasize Rax Cloud Serves API and try to leave Nova out of it. If i'm wrong, if it is more truthful/useful; to say Nova, i am happy to make that change.